src/internal/modules/billing/routes/bill-run.js

```diff
@@ -235,7 +237,9 @@ const routes = {
       validate: {
         params: Joi.object().keys({
           batchId: Joi.string().uuid(),
-          invoiceId: Joi.string().uuid()
+          invoiceId: Joi.string().uuid(),
+          originalInvoiceId: Joi.string().uuid(),
+          rebillInvoiceId: Joi.string().uuid()
```

Comment:
same as above you should remove the last part of the route and add the below as query parameters 
```
originalInvoiceId: Joi.string().uuid().optional(),
rebillInvoiceId: Joi.string().uuid().optional()
```